sources/packages/web/src/types/contracts/StudentContract.ts

```diff
@@ -33,3 +34,17 @@ export enum FileOriginType {
   Student = "Student",
   Ministry = "Ministry",
 }
+
+/**
+ * Contract for student restriction.
+ */
+export interface StudentRestriction {
```

Comment:
Is this a DTO? If it is used only for the state, please place it alongside the other state types.